src/pages/myGeotab/guides/codeBase/usingInDotnet.tsx

```diff
@@ -0,0 +1,164 @@
+import { ReactNode } from "react";
+import { Page } from "../../../../components";
+import { PageTitleProps } from "../../../../components/PageTitle/PageTitle";
+import { HeaderSections } from "../../../../components/Header/headerSectionsEnum";
+import Accordion from "../../../../components/Accordion/Accordion";
+import { TableOfContentsItem } from "../../../../components/TableOfContents/TableOfContents";
+import InformationalBox from "../../../../components/InformationalBox/InformationalBox";
+import CodeSample from "../../../../components/CodeSamplesContainer/CodeSample";
+
+const nugetPackage: ReactNode = <div>
+    The <a href="https://www.nuget.org/packages/Geotab.Checkmate.ObjectModel/" target="_blank" rel="noopener noreferrer">nuget package</a> is an SDK library for accessing MyGeotab customer databases. 
+    It is a convenient "wrapper" around Geotab's HTTP/JSON API to allow developers focus on writing code instead of moving data over the wire. It includes tools to assist authenticating against Geotab's 
+    servers, automatically serializing/deserializing JSON, and providing definitions for Checkmate object classes.
+    <br></br>
+    Please remember to regularly check for Geotab.Checkmate.ObjectModel nuget package updates. Ideally, your integration should use the same Geotab.Checkmate.ObjectModel nuget package version
+    as the one that your database is on.
+    <InformationalBox>
+        <p>Quick start in <a href="../../api/clients">{/*TODO: fix link*/}API Clients</a></p>
+    </InformationalBox>
+</div>
+
+const apiClass: ReactNode = <div>
+    <h2>Step 1: Initialize & Authentication</h2>
+    <p>In order to have access to the API class, add the following to the file's includes section:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `using Geotab.Checkmate;
+using Geotab.Checkmate.ObjectModel;`
+        }
+    ></CodeSample>
+    <p>An instance of API can now be constructed to be used in the code. For the most basic use case, all the data that is needed is user credentials and a database name:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `var api = new API(userName, password, null, databaseName, server);`
+        }
+    ></CodeSample>
```

Comment:
This can be self-closing since there is no content inside the tags.
```suggestion
    />
```